runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessGlobalVariables.ts

```diff
@@ -0,0 +1,69 @@
+import { is } from 'bpmn-js/lib/util/ModelUtil';
+
+import { useSelector } from '#src-app/store';
+import { globalVariableSelector } from '#src-app/store/slices/GlobalVariable';
+import { BPMNElement, CamundaInputOutputElement } from '#src-app/views/process/ProcessBuildView/Modeler/helpers/elementParameters';
+
+import { useModelerContext } from './useModelerContext';
+
+const useProcessGlobalVariables = () => {
+    const { globalVariables } = useSelector(globalVariableSelector);
+    const context = useModelerContext();
+
+    const assignVariablesElements =
+                context?.modeler
```

Comment:
keep `context?.modeler` in the line above then those chains will be shifted to the left